gcloud/streaming/transfer.py

```diff
@@ -35,51 +34,99 @@
 RESUMABLE_UPLOAD = 'resumable'
 
 
+_DEFAULT_CHUNKSIZE = 1 << 20
```

Comment:
It seems we've reached an impasse. What do you say about my contention about the "average" maintainer?